data/clients.yaml

```diff
@@ -121,8 +121,9 @@ featureGroups:
     - name: env_JAEGER_PROPAGATION
       description: >
         `JAEGER_PROPAGATION` defines the propagation format used by the tracer.
-        Supported values are `jaeger` (defined [here](../client-libraries/#propagation-format))
-        and `b3` (defined [here](https://github.com/openzipkin/b3-propagation)).
+        Supported values are `jaeger` (defined [here](../client-libraries/#propagation-format)),
+        `b3` (defined [here](https://github.com/openzipkin/b3-propagation))
+        and `trace_context` (defined [here](https://github.com/w3c/distributed-tracing))
```

Comment:
Only comment is whether `trace_context` is a common/consistent value used across jaeger tracers (or other projects)? Wondering if `w3c` may be more concise?